apps/web/src/server/views/applications/case-documentation/search-results/document-search.component.njk

```diff
@@ -0,0 +1,29 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% macro applicationDocumentSearch(params) %}
+
+	{% set csrfToken = params.csrfToken %}
+
+	<form method="post" action="project-documentation/document-search-results" novalidate class="pins-dashboard-box--form pins-dashboard-form" role="search" aria-label="Applications">
+		{% include 'app/includes/csrf.njk' %}
+		{{ govukInput({
+			label: {
+				text: params.label,
+				classes: "pins-dashboard-form--label",
+				isPageHeading: false
+			},
+			errorMessage: params.errors.query | errorMessage,
```

Comment:
here you're expecting some error, but this page is never rendered by the controller in case of errors